packages/IMAPClient-Core.package/ICEmail.class/instance/deserializeFrom..st

```diff
@@ -0,0 +1,8 @@
+accessing-header
+deserializeFrom: aJson
+
+	self header: (Dictionary newFrom: (aJson at: 'header' ifAbsent: [JsonObject new]));
```

Comment:
Maybe that kind of initialization is worth a mention in the class comment?

Reply:
@JuliaJoch